core/src/main/scala-3/cats/derived/package.scala

```diff
@@ -0,0 +1,44 @@
+package cats.derived
+
+import cats.*
+
+object semiauto extends
+  CommutativeMonoidDerivation,
+  CommutativeSemigroupDerivation,
+  ContravariantDerivation,
+  EmptyDerivation,
+  EmptyKDerivation,
+  EqDerivation,
+  FoldableDerivation,
+  FunctorDerivation,
+  HashDerivation,
+  InvariantDerivation,
+  MonoidDerivation,
+  MonoidKDerivation,
+  OrderDerivation,
+  PartialOrderDerivation,
+  ReducibleDerivation,
+  SemigroupDerivation,
+  SemigroupKDerivation,
+  ShowDerivation,
+  TraverseDerivation,
+  Instances:
+
+  def foldable[F[_]](using ev: DerivedFoldable[F]): Foldable[F] = ev.instance
+
+  def functor[F[_]](using ev: DerivedFunctor[F]): Functor[F] = ev.instance
+
+  def reducible[F[_]](using ev: DerivedReducible[F]): Reducible[F] = ev.instance
+
+  def traverse[F[_]](using ev: DerivedTraverse[F]): Traverse[F] = ev.instance
```

Comment:
Those should be inline similar to the extension methods

Reply:
Sorry, I probably need to look at the opaque type change in more detail. Why is `inline` necessary?

Reply:
Because `type DerivedFoldable[F[_]] = Derived[Foldable[F]]` and `object DerivedFoldable` are not actually companions and you would have to `import DerivedFoldable.given` - but the inline method does a trick and imports this scope before calling `summonInline[DerivedFoldable[F]]`.

Reply:
Ah! 👍 thanks